src/pages/EventDetail/EventDetail.jsx

```diff
@@ -1,7 +1,153 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { S } from './EventDetail.js';
+import Chart from './Chart.jsx';
+import Map from './Map.jsx';
+import EventCard from '../../components/EventCard/EventCard.jsx';
 
 const EventDetail = () => {
-  return <div>EventDetail</div>;
+  const [eventData, setEventData] = useState([]);
+  const [detail, setDetail] = useState([]);
+  const [btnText, setBtnText] = useState('기부하고 바로 구매');
+  const [bidToken, setBidToken] = useState([]);
+
+  const titleMapping = {
+    location: '장소',
+  };
+
+  // Object.entries(detail).map(([key, value])
+
+  useEffect(() => {
+    fetch('/data/Graphdata.json')
+      .then(response => response.json())
+      .then(result => {
+        setBidToken(result);
+        console.log(result);
+      });
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/DetailEventSample.json')
+      .then(response => response.json())
+      .then(result => setEventData(result));
+  }, []);
+
+  useEffect(() => {
+    fetch('/data/DetailSample.json')
+      .then(response => response.json())
+      .then(result => {
+        setDetail(result[0]);
+      });
+  }, []);
+
+  const handleMouse = isMouseOver => {
+    if (isMouseOver) {
+      setBtnText(`${detail.highest_events_token}토큰`);
+    } else {
+      setBtnText('기부하고 바로 구매');
+    }
+  };
+
+  const MOCK_DATA = [
+    {
+      title: '장소',
+      valeue: detail.location,
```

Comment:
```suggestion
      value: detail.location,
```
키 값 이름이 잘 못 된 것 같네요!